astro/src/pages/volunteer.astro

```diff
@@ -0,0 +1,156 @@
+          <label class="form-check-label ps-2 display-65" for="interestWrite">
+            Writing Content
+          </label>
+        </div>
+        <div class="form-check d-flex mb-2 align-items-center">
+          <input class="form-check-input check" type="checkbox" name="flexRadioDefault" id="interestOther" aria-checked="false">
+          <label class="form-check-label ps-2 display-65" for="interestOther">
+            Other
+          </label>
+        </div>
+      </fieldset>
+    </div>
+    <div class="col-md-6 mb-3">
+      <fieldset>
+        <legend class="mb-2 display-65">
+          How Many Hours Per Week Would You Like to Volunteer? <abbr class="display-65 text-ac" title="required">*</abbr>
+        </legend>
+        <div class="form-check d-flex mb-2 align-items-center">
+          <input class="form-check-input check" type="radio" name="hoursLess" id="hoursLess" aria-checked="false">
+          <label class="form-check-label ps-2 display-65" for="hoursLess">
+            Less than 4 hours a week
+          </label>
+        </div>
+        <div class="form-check d-flex mb-2 align-items-center">
+          <input class="form-check-input check" type="radio" name="hoursAround" id="hoursAround" aria-checked="false">
+          <label class="form-check-label ps-2 display-65" for="hoursAround">
+            Around 4 hours a week
+          </label>
+        </div>
+        <div class="form-check d-flex mb-2 align-items-center">
+          <input class="form-check-input check" type="radio" name="hoursMore" id="hoursMore" aria-checked="false">
+          <label class="form-check-label ps-2 display-65" for="hoursMore">
+            More than 4 hours a week
+          </label>
+        </div>
+      </fieldset>
+    </div>
+    <div class="form-floating col-md-6 mb-3">
+      <label for="accommodateTextarea" class="form-label mb-1 display-65">Please list any accomodations needed for remote meetings and work.</label>
+      <textarea class="form-control w-100 form-text-area" id="accomodateTextarea"></textarea>
+    </div>
+    <div class="row">
+    <div class="col">
+      <button type="submit" class="btn bg-ac text-white display-65 w-100 volunteer-button">Submit</button>
+    </div>
+  </div>
+</form>
+
+<style>
+  /* first declaration that is commented out below is for debugging purposes. */
+  /* * {
+    outline: 1px solid black;
+  } */
+
+  *:focus-visible, .form-field:focus {
+    outline: 6px dashed #041058;
+    outline-offset: 0px;
+    box-shadow: none;
+    border-radius: 0.5rem;
+  }
+
+  .volunteer-form {
+    border: 3.5px solid;
+    border-radius: 0.5rem;
+  }
+
+  .check {
+   width: 30px;
+   height: 30px;
+  }
+
+  input[type=checkbox], input[type=radio] {
+    accent-color: #041058;
+  }
+
+  input[type=checkbox]:hover, input[type=radio]:hover {
+    border-color: #041058;
+  }
+
+  fieldset label {
+    cursor: pointer;
+  }
+
+  .form-text-area {
```

Comment:
Let's use `em` instead of pixels to set the height. This will be more consistent across browsers.